index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio of Utsargo Roy</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/c488aa6c77.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="navbar col-md-12 space-between align-middle">
+            <h3 class="siteTitle">Welcome</h3>
+            <i class="fa-solid fa-bars"></i>
+        </div>
+        <section class="welcome display-flex flex-column space-arround align-middle">
+            <h2 class="greetings txt-center neutrals">Hey there. I'm Utsargo</h2>
+            <h2 class="greetings txt-center success">I'm a micronaut at Microverse</h2>
+            <p class="txt-center neutrals">
+                I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+            </p>
+            <div class="socialLinks col-md-12 display-flex flex-row space-around align-middle">
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-solid fa-m"></i>
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-angellist"></i>
+            </div>
```

Comment:
- [ ] This is a list of social links for your social media pages. The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly introduce those tags in this list of social links.
 - [ ] The list of icons should have a row direction like the Figma template, but it has a column on the mobile view. Please change that and add some spacing between each icon.
 - Your design:
 - 
<img width="490" alt="image" src="https://user-images.githubusercontent.com/61919617/198047309-36e34d97-be40-4915-946c-d00d01d63513.png">

 - Figma template:
 - 
<img width="263" alt="image" src="https://user-images.githubusercontent.com/61919617/198047393-c37cf8ff-e1ea-40cc-bb54-a1b906a091f4.png">